pkgs/development/interpreters/python/meta-package.nix

```diff
@@ -0,0 +1,58 @@
+{
+  buildPythonPackage,
+  lib,
+  hatchling,
+}:
+{
+  pname,
+  version,
+  dependencies ? [ ],
+  optional-dependencies ? { },
+  passthru ? { },
+  meta ? { },
```

Comment:
Inheriting `meta` doesn't make sense for the license, does it?

Reply:
That's a bit of a philosophical question :sweat_smile: 

Either we consider it as `psycopg2-binary` functionality provided by `psycopg2`, under the license terms of `psycopg2`.
Or we leave it empty, just inheriting homepage & description. I'm OK with either.